Examples/src/main/res/values/developer-config.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="mapbox_access_token">pk.eyJ1IjoiZGVuaXNrb3JvbmNoaWstZGV2IiwiYSI6ImNqdHNqMTRsbjBwaWI0ZnFudTVzbWl6bXQifQ.hUcnwjXhZAMHO5leM8wSCw</string>
```

Comment:
are you sure, that token should be in common access?

Reply:
Definitely no, I'll remove it and we'll rotate the token now, too.Nice catch!